tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs

```diff
@@ -40,6 +41,13 @@ public UtilitiesMainPageViewModel()
                 NavigateUri = "https://aka.ms/PowerToysOverview_EnvironmentVariables",
                 ImageSource = Path.Combine(AppContext.BaseDirectory, "Assets\\EnvironmentVariables", "EnvironmentVariables.ico"),
             },
+            new(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), $"Microsoft\\WindowsApps\\{Package.Current.Id.FamilyName}\\devhome.pi.exe"), experimentationService, "ProjectIronsidesExperiment")
+            {
+                Title = stringResource.GetLocalized("ProjectIronsidesTitle"),
+                Description = stringResource.GetLocalized("ProjectIronsidesDesc"),
+                NavigateUri = "https://aka.ms/projectironsides",

```

Comment:
We use go links elsewhere in Dev Home, I think they're preferred to aka.ms but PM would know.

Reply:
I'll leave this as aka.ms/ for now, the documentation isn't ready yet and I'll follow up over email.